src/core/RenderTarget.js

```diff
@@ -35,6 +35,7 @@ class RenderTarget extends EventDispatcher {
 	 * @property {?Texture} [depthTexture=null] - Reference to a depth texture.
 	 * @property {number} [samples=0] - The MSAA samples count.
 	 * @property {number} [count=1] - Defines the number of color attachments . Must be at least `1`.
+	 * @property {number} [depth=1] - The texture depth.
```

Comment:
I was thinking about changing the ctor signature to the following since it looks more consistent:
```js
const renderTarget = new RenderTarget( width, height, depth, options );
```
But I'm afraid this will heavily affect existing code since most use cases just use the default depth value. Maybe it's indeed better to store the depth in the options instead...

Reply:
`RenderTarget ` is a very sensible Class heavily used by the community I would definitely try to avoid changing the constructor signature and use options instead.